echopype/convert/set_groups_base.py

```diff
@@ -25,6 +26,7 @@ def __init__(
         compress=True,
         overwrite=True,
         params=None,
+        parser2zarr_obj=None,
```

Comment:
Again here with the tense, shouldn't it be `parsed2zarr` instead?

Reply:
As noted in a previous comment, this was a small oversight on my part. I will correct all occurrences to `parsed2zarr`.